Automattic/jetpack
# Social | Initial state: Migrate useEditorPreview feature flag to script data

Extracted out of #38952, this PR migrates on the `useEditorPreview` feature flag and other feature flags will be migrated in the follow up PRs.






## Proposed changes:

* Migrate `useEditorPreview` feature flag to script data
* Clean up social store of feature flags

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On Simple and Atomic sites, goto post editor
* Confirm that the editor loads fine
* Open Jetpack sidebar
* Confirm that "Preview social posts" button is not visible
* On a Jetpack site, goto post editor
* Open Jetpack/Social sidebar
* Confirm that "Preview social posts" is visible and opens the modal
